docs/user_docs/maintaince/scale/horizontal-scale.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Horizontal Scale
+description: How to horizontally scale a cluster
+keywords: [horizontal scale, horizontal scaling]
+sidebar_position: 1
+sidebar_label: Horizontal Scale
+---
+
+# Horizontal scaling
+
+Horizontal scaling changes the amount of pods. For example, you can apply horizontal scaling to scale pods up from three to five. The scaling process includes the backup and restore of data.
+
+## Before you start
+
+Check whether the cluster STATUS is `Running`. Otherwise, the following operations may fail.
+
+```bash
+kbcli cluster list mycluster
+>
+NAME        NAMESPACE   CLUSTER-DEFINITION   VERSION        TERMINATION-POLICY   STATUS    CREATED-TIME
+mycluster   default     mysql                mysql-8.0.33   Delete               Running   Jul 05,2024 18:46 UTC+0800
+```
+
+## Scale replicas
+
+### Steps
+
+1. Change configuration.
+
+    Configure the parameters `--components` and `--replicas`, and run the command.
+
+    ```bash
+    kbcli cluster hscale mycluster \
+    --components="mysql" --replicas=3
+    ```
+
+    - `--components` describes the component name ready for horizontal scaling.
+    - `--replicas` describes the replica amount of the specified components.
+
+2. Validate the horizontal scaling operation.
+
+    Check the cluster STATUS to identify the horizontal scaling status.
+
+    ```bash
+    kbcli cluster list mycluster
+    ```
+
+    - STATUS=Updating: it means horizontal scaling is in progress.
+    - STATUS=Running: it means horizontal scaling has been applied.
+
+3. Check whether the corresponding resources change.
+
+    ```bash
+    kbcli cluster describe mycluster
+    ```
+
+## Scale instances
+
+### Why do you need to scale for specified instances
+
+KubeBlocks generated workloads as *StatefulSets*, which was a double-edged sword. While KubeBlocks could leverage the advantages of a *StatefulSets* to manage stateful applications like databases, it inherited its limitations.
```

Comment:
From v0.9, KubeBlocks no longer use StatefulSets, could ask @free6om for help with these materials.